pkg/build/builder/docker.go

```diff
@@ -63,13 +57,13 @@ func (d *DockerBuilder) Build() error {
 	if err != nil {
 		return err
 	}
-	if err = d.fetchSource(buildDir); err != nil {
+	if err := fetchSource(buildDir, d.build, d.urlTimeout, os.Stdin, d.git); err != nil {
 		return err
 	}
-	if err = d.addBuildParameters(buildDir); err != nil {
+	if err := d.addBuildParameters(buildDir); err != nil {
 		return err
 	}
-	glog.V(4).Infof("Starting Docker build from %s/%s BuildConfig ...", d.build.Namespace, d.build.Name)
+	glog.V(4).Infof("Starting Docker build from build config %s ...", d.build.Name)
```

Comment:
Builders are human out, anyone looking at the log has to be in a namespace to find it.